contactaccessrights/CRM/Contactaccessrights/Test/Fabricator/Rights.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Fabricates rights for tests.
+ */
+abstract class CRM_Contactaccessrights_Test_Fabricator_Rights {
+
+  /**
+   * Fabricates Rights, given specific values for contact_id, entity_id and 
+   * entity_type.
+   * 
+   * @param array $params
+   *   Associative array of parameters to be used to store rights infrmation in 
+   *   database.
+   * @return array
+   *   Holds values stored in database for created acess right
+   * @throws Exception

```

Comment:
Since this method now only throws CiviCRM_API3_Exceptions, you can remove this @throws